config/prometheus.yaml

```diff
@@ -69,6 +69,15 @@ data:
           duration: $PROMETHEUS_NODE_NOT_READY_WAIT_FOR
         annotations:
           summary: "AlertManager has detected Node(s) in an unready state."
+      - alert: PodRestartAlert
+        expr: increase(kube_pod_container_status_restarts_total[5m]) > 5
+        for:  $PROMETHEUS_POD_NOT_HEALTHY_WAIT_FOR
+        labels:
+          severity: warning
+        annotations:
+          summary: "AlertManager has detected Pods in multiple restarting state.""
```

Comment:
This looks like it should be a syntax error with the multiple ending double quotes - is it deploying correctly for you?